src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -432,7 +456,7 @@ private static string GetTraceBack(Exception e)
         ///     Data Marshaler for all data coming into a Python node.
         /// </summary>
         [SupressImportIntoVM]
-        public static DataMarshaler InputMarshaler
+        public override object InputDataMarshaler
```

Comment:
It needs to replace an existing public API
PythonEngine.InputDataMarshaler will be used instead of the already existing public API CPythonEvaluator.InputMarshaler